Config/FilterFactoryCollection.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+/*
+ * This file is part of the `liip/LiipImagineBundle` project.
+ *
+ * (c) https://github.com/liip/LiipImagineBundle/graphs/contributors
+ *
+ * For the full copyright and license information, please view the LICENSE.md
+ * file that was distributed with this source code.
+ */
+
+namespace Liip\ImagineBundle\Config;
+
+use Liip\ImagineBundle\Exception\Config\Filter\NotFoundException;
+use Liip\ImagineBundle\Factory\Config\FilterFactoryInterface;
+
+final class FilterFactoryCollection implements FilterFactoryCollectionInterface
+{
+    /**
+     * @var FilterFactoryInterface[]
+     */
+    private $filterFactories;
+
+    /**
+     * FilterCollection constructor.
+     *
+     * @param FilterFactoryInterface ...$filterFactories
+     */
+    public function __construct(FilterFactoryInterface ...$filterFactories)
+    {
+        $this->filterFactories = $filterFactories;
```

Comment:
imho we should convert this to a hashmap with name => factory so that we don't have to loop over the array in getFilterFactoryByName

Reply:
done